include/powerloader/context.hpp

```diff
@@ -35,8 +35,10 @@ namespace powerloader
     class mirror_map_type : private mirror_map_base
     {
     public:
+        using mirror_map_base::begin;
```

Comment:
I initially voluntarilly didnt expose `begin/end` because that breaks the garantee that the values of the vector are unique (because then we can access and modify the vectors).
Suggestions: either
1. use `as_map()` which makes a copy, not modifying this container;
2. or expose the const versions of `begin/end` only.